packages/core/docs/commercetools/use-facet.md

```diff
@@ -0,0 +1,49 @@
+# useFacet composable
+
+A composable responsible for (basic) faceting. This implementation rely on GraphQL api which means it not 100% complete faceting as commercetools provide in their dedicated rest-based faceting api. This one based on product and category query provided by `getProduct` and `getCategory` functions from `@vue-storefront/commercetools-api`.
+
+## Search input params
+
+```ts
+interface ProductsSearchParams {
+  perPage?: number;
+  page?: number;
+  sort?: any;
+  term?: any;
+  filters?: Record<string, Filter>;
+  catId?: string | string[];
+  skus?: string[];
+  slug?: string;
+  id?: string;
+}
+```
+
+## Examples
+
+Using a full working example of the category page browsing.
+
+```js
+setup(props, context) {
+  const { result, search, loading } = useFacet();
+  const products = computed(() => facetGetters.getProducts(result.value));
```

Comment:
list of getters should be part of a docs, not an example, also you should either describe what they're returning or paste typings